packages/devtools_app/lib/src/screens/memory/memory_instance_tree_view.dart

```diff
@@ -94,8 +94,8 @@ class _FieldTypeColumn extends TreeColumnData<FieldReference> {
 
   @override
   int compare(FieldReference a, FieldReference b) {
-    final Comparable valueA = getValue(a);
-    final Comparable valueB = getValue(b);
+    final Comparable valueA = getValue(a) as Comparable<Object>;
+    final Comparable valueB = getValue(b) as Comparable<Object>;
```

Comment:
these casts should be able to be removed once getValue return type is changed to string. here and below.

Reply:
done